code/modules/research/xenoarchaeology/artifact/artifact.dm

```diff
@@ -124,6 +124,15 @@ var/list/all_generated_artifact_ids = list()
 					var/spawn_type = artifact_find.artifact_find_type
 					if (spawn_type == /obj/machinery/artifact)
 						new spawn_type(get_turf(src), artifact_find.artifact_id)
+					else if (spawn_type == /obj/machinery/power/supermatter)
+						spawn(rand(10 MINUTES, 30 MINUTES))//The time it takes for Nanotrasen to detect it and make the Science dept an offer they cannot refuse.
+							var/already = 0
+							for(var/datum/centcomm_order/O in SSsupply_shuttle.centcomm_orders)
```

Comment:
Might as well do a locate(/datum/centcomm_order/department/science/supermatter) in SSsupply_shuttle.centcomm_orders